tests/page_test.go

```diff
@@ -336,6 +350,10 @@ func TestPageInputSpecialCharacters(t *testing.T) {
 }
 
 func TestPageFill(t *testing.T) {
+	// these tests are not parallel by intention because
+	// they're testing the same page instance and they're
+	// faster when run sequentially.
```

Comment:
Can you clarify this please? How did you measure this? At least with my silly tests this does not seem to be the case :thinking:

Reply:
Subtests race with each other since they change the same page content. I didn't bother making only the top-level test parallel, but not the subtests. But we would have to add a `nolinter` direction. Since we have many tests, running this sequentially didn't make a difference.

On the other hand, even if we make each subtest to create and run their own browsers (and not share them), the test run for this test will take 2X. On my machine, it takes 7.5s.

Reply:
I've improved the comment: https://github.com/grafana/xk6-browser/pull/1019/commits/122818fc3d0bd754d8fb563e49b080406fb55d5f.

Reply:
Right, I missed that there are two subests that use the same selector. Does not make much sense that the linter only complains if you run the top level test in parallel and not the subtests, but not if you just don't run the top level in parallel :man_shrugging: 
Anyway, we can leave it like this :+1:

Reply:
Linters :)